src/Tribe/Tickets_Handler.php

```diff
@@ -1044,7 +1053,7 @@ public function save_global_stock( $post_id ) {
 	 *
 	 */
 	public function save_tickets_order( $post_id ) {
-		if ( ! ( isset( $_POST[ 'tribe-tickets-post-settings' ] ) && wp_verify_nonce( $_POST[ 'tribe-tickets-post-settings' ], 'tribe-tickets-meta-box' ) ) ) {
```

Comment:
As far as I can tell we should still be checking if there is a nonce here. We wouldn't want to save the order without one, right?